.changeset/lucky-llamas-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mysten/ledgerjs-hw-app-sui": major
```

Comment:
Probably make the initial release a minor to avoid lots of breaking major versions as we iterate